<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>SeeSig</string>
  </property>
  <property name="windowIcon">
   <iconset resource="SeeSigResources.qrc">
    <normaloff>:/SeeSigResource/RESOURCES/SeeSig_icon_floating.png</normaloff>:/SeeSigResource/RESOURCES/SeeSig_icon_floating.png</iconset>
  </property>
  <widget class="QWidget" name="main">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="8,1,0">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="acceptDrops">
       <bool>true</bool>
      </property>
      <property name="currentIndex">
       <number>-1</number>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="tool_signal_analysis">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string>Signal Analysis</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="signal_average">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Signal Average</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="signal_average_label_value">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="1" rowspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="signal_first_harmonic">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Signal first harmonic</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="signal_first_harmonic_label_value">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="signal_std_deviation">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Signal Standard Deviation</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="signal_std_label_value">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="signalCalculator">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="title">
       <string>Signal Calculator</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="op1_label">
         <property name="text">
          <string>Op 1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="op1_edit">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="operation_symbol">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <item>
          <property name="text">
           <string>X</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>/</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>+</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>-</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="op2_label">
         <property name="text">
          <string>Op 2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="op2_edit">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="show_result_operation">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Show</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="tearOffEnabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionFile_format"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSettings"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionreset_zoom"/>
   <addaction name="actionzoom_rect"/>
   <addaction name="actionnew_plot"/>
   <addaction name="actionopen_plot"/>
   <addaction name="actionsave_plot"/>
   <addaction name="actionone_graph_per_plot"/>
   <addaction name="actionnormalize"/>
   <addaction name="actionmouse_cursors_at_signal"/>
   <addaction name="actionauto_signal_analysis"/>
   <addaction name="action_signal_calculator"/>
   <addaction name="actionset_label"/>
   <addaction name="actionremove_label"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionreset_zoom">
   <property name="icon">
    <iconset resource="SeeSigResources.qrc">
     <normaloff>:/SeeSigResource/RESOURCES/default_zoom.PNG</normaloff>:/SeeSigResource/RESOURCES/default_zoom.PNG</iconset>
   </property>
   <property name="text">
    <string>reset_zoom</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Set the zoom to 0&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
  <action name="actionnew_plot">
   <property name="icon">
    <iconset>
     <normalon>:/SeeSigResource/RESOURCES/new_plot.PNG</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>new_plot</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Open a new plot&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
  <action name="actionopen_plot">
   <property name="icon">
    <iconset>
     <normalon>:/SeeSigResource/RESOURCES/open_plot.PNG</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>open_plot</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Open signal data&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
  <action name="actionsave_plot">
   <property name="icon">
    <iconset>
     <normalon>:/SeeSigResource/RESOURCES/save_plotting.PNG</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>save_plot</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Save current display plots&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
  <action name="actionone_graph_per_plot">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/SeeSigResource/RESOURCES/1_graph_per_plot.PNG</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>one_graph_per_plot</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Draw each signal on a different plot&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
  <action name="actionnormalize">
   <property name="icon">
    <iconset resource="SeeSigResources.qrc">
     <normaloff>:/SeeSigResource/RESOURCES/normalize.png</normaloff>
     <normalon>:/SeeSigResource/RESOURCES/normalize.png</normalon>
     <disabledoff>:/SeeSigResource/RESOURCES/normalize.png</disabledoff>
     <disabledon>:/SeeSigResource/RESOURCES/normalize.png</disabledon>
     <activeoff>:/SeeSigResource/RESOURCES/normalize.png</activeoff>
     <activeon>:/SeeSigResource/RESOURCES/normalize.png</activeon>
     <selectedoff>:/SeeSigResource/RESOURCES/normalize.png</selectedoff>
     <selectedon>:/SeeSigResource/RESOURCES/normalize.png</selectedon>:/SeeSigResource/RESOURCES/normalize.png</iconset>
   </property>
   <property name="text">
    <string>normalize</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Normalize by maximum value&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
  <action name="actionFile_format">
   <property name="text">
    <string>File format...</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open..</string>
   </property>
  </action>
  <action name="actionmouse_cursors_at_signal">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="SeeSigResources.qrc">
     <normaloff>:/SeeSigResource/RESOURCES/mouse_cursors_at_signal.PNG</normaloff>:/SeeSigResource/RESOURCES/mouse_cursors_at_signal.PNG</iconset>
   </property>
   <property name="text">
    <string>mouse_cursors_at_signal</string>
   </property>
   <property name="toolTip">
    <string>Set bars at signals</string>
   </property>
  </action>
  <action name="actionzoom_rect">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="SeeSigResources.qrc">
     <normaloff>:/SeeSigResource/RESOURCES/band_zoom.PNG</normaloff>:/SeeSigResource/RESOURCES/band_zoom.PNG</iconset>
   </property>
   <property name="text">
    <string>zoom_rect</string>
   </property>
   <property name="toolTip">
    <string>Zoom into a defined rectangle at plot</string>
   </property>
  </action>
  <action name="actionauto_signal_analysis">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="SeeSigResources.qrc">
     <normaloff>:/SeeSigResource/RESOURCES/auto_signal_analysis.ico</normaloff>:/SeeSigResource/RESOURCES/auto_signal_analysis.ico</iconset>
   </property>
   <property name="text">
    <string>auto_signal_analysis</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About...</string>
   </property>
  </action>
  <action name="action_signal_calculator">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="SeeSigResources.qrc">
     <normaloff>:/SeeSigResource/RESOURCES/signal_calculator.PNG</normaloff>:/SeeSigResource/RESOURCES/signal_calculator.PNG</iconset>
   </property>
   <property name="text">
    <string>signal_calculator</string>
   </property>
   <property name="toolTip">
    <string>Signal calculator</string>
   </property>
  </action>
  <action name="actionset_label">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="SeeSigResources.qrc">
     <normaloff>:/SeeSigResource/RESOURCES/set_label_tool.PNG</normaloff>:/SeeSigResource/RESOURCES/set_label_tool.PNG</iconset>
   </property>
   <property name="text">
    <string>set_label</string>
   </property>
   <property name="toolTip">
    <string>Insert a label at the plot</string>
   </property>
  </action>
  <action name="actionremove_label">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="SeeSigResources.qrc">
     <normaloff>:/SeeSigResource/RESOURCES/remove_label_tool.PNG</normaloff>:/SeeSigResource/RESOURCES/remove_label_tool.PNG</iconset>
   </property>
   <property name="text">
    <string>remove_label</string>
   </property>
   <property name="toolTip">
    <string>Remove the selected label</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="SeeSigResources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionOpen</sender>
   <signal>triggered()</signal>
   <receiver>actionopen_plot</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
